http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20200915_add_chart_to_app_and_cluster_tables.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="leonardo" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet logicalFilePath="leonardo" author="kyuksel" id="add_chart_to_app_table">
+        <addColumn tableName="APP">
+            <column name="chart" type="VARCHAR(254)">
+                <constraints nullable="true"/>
+            </column>
+        </addColumn>
+
+        <sql>UPDATE APP SET chart = 'dummyChartName-x.y.z' WHERE chart IS NULL</sql>
```

Comment:
Maybe we should set existing records to the actual value, e.g. `galaxy/galaxykubeman-0.5.3`?

Reply:
I was talking to Qi about this too. Thought using the dummy value may have two advantages:

1. I'm guessing we have some records that were created with older versions of `galaxykubeman`, and possibly `nginx` so the latest value may be wrong for them.
2. It could be good to know what records were created before the columns were added for debug purposes.

What do you think?

Reply:
Sure, that sounds reasonable